signwriting_animation/data/segment_poses.py

```diff
@@ -0,0 +1,52 @@
+# get_data.py
+import os
+import pandas as pd
+import argparse
+
+def create_fixed_segments(data_csv, output_csv, window_size=60):
```

Comment:
there's no need to create segments and save them on disk.
You can do this on the fly - load the pose file, then randomally choose a frame to cut until for context, and cut from for the generated data. using fixed windows might not be beneficial here